js/miniature.js

```diff
@@ -0,0 +1,19 @@
+import {getSimilarObjects} from './data.js';
+
+const fragment = document.createDocumentFragment('.pictures');
+const pictureTemplate = document.querySelector('#picture').content;
+
+
+const similarObjects = getSimilarObjects();
```

Comment:
```suggestion
const similarMiniatures = getSimilarMiniatures();
```

Давай сделаем нейминг чуть более понятным и заменим слово объекты на картинки или миниатюры?